migrations/AttendanceIncentivizer.sol

```diff
@@ -0,0 +1,87 @@
+pragma solidity ^0.4.11;
+
+contract AttendanceIncentivizer {
+  event Donate(address indexed donater, uint indexed amount);
+  event Payout(address indexed donater, uint indexed amount);
+  event Validate(address indexed validator, address indexed validatee);
+
+  mapping (address => uint) public donaters;
+  mapping (address => uint) public validations;
+  address[] public donaterList;
+  address public owner;
+  address public winner;
```

Comment:
couldn't winner be a local variable in the calculateWinner function?

Reply:
Agree.